playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -42,12 +42,27 @@
     retries: 60
     delay: 60
 
+    # Run the pre-upgrade hook if defined:
+  - debug: msg="Running node pre-upgrade hook {{ openshift_node_upgrade_pre_hook }}"
+    when: openshift_node_upgrade_pre_hook is defined
+
+  - include_tasks: "{{ openshift_node_upgrade_pre_hook }}"
+    when: openshift_node_upgrade_pre_hook is defined
+
   post_tasks:
   - include_role:
       name: openshift_node
       tasks_from: upgrade.yml
     vars:
       openshift_node_upgrade_in_progress: True
+  
```

Comment:
@jtudelag this line contains a trailing space which is causing the linter to fail